lib/externalGradingSocket.sql

```diff
@@ -0,0 +1,32 @@
+-- BLOCK select_submissions_for_variant
```

Comment:
If there are multiple grading_jobs for a submission then we'll get multiple rows for a single submission. This could be fixed with:
```sql
SELECT DISTINCT ON (s.id)
```
and adding at the end:
```sql
ORDER BY s.id, gj.id DESC
```
(to get the grading job with the largest ID if there is more than one of them).